ixmp/backend/base.py

```diff
@@ -882,15 +882,17 @@ def get_meta(self, s: Scenario):
         """
 
     @abstractmethod
-    def set_meta(self, s: Scenario, name, value):
-        """Set a single metadata key.
+    def set_meta(self, s: Scenario, name_or_data, value=None):
+        """Set single or multiple metadata entries.
```

Comment:
please rename to `meta` throughout

Reply:
updated (I hope I didn't miss any other places)